### PR TITLE
Update Chicken.php

### PR DESCRIPTION
**Pull request by erkam2002**
Update Chicken.php now can chicken move

### CODE REVIEW

File: src/pocketmine/entity/Chicken.php [hunk 1]
```diff
@@ -24,44 +24,58 @@
  *
 */
 
-namespace pocketmine\entity;
+<?php
```
Comment (kiyoshi-kurosaki) [inline src/pocketmine/entity/Chicken.php]: Need to remove that line :D

**Comment by keithkfng** [comment]
Mob AI was never meant to move - it lags the whole server down.

File: src/pocketmine/entity/Chicken.php [hunk 2]
```diff
@@ -24,44 +24,58 @@
...
+
+namespace milk\entitymanager\entity;
```
Comment (thebigsmileXD) [inline src/pocketmine/entity/Chicken.php]: One more issue. Incorrect name space.

File: src/pocketmine/entity/Chicken.php [hunk 3]
```diff
@@ -24,44 +24,58 @@
...
 
-use pocketmine\event\entity\EntityDamageEvent;
-use pocketmine\item\Item as drp;
+use pocketmine\item\Item;
```
Comment (thebigsmileXD) [inline src/pocketmine/entity/Chicken.php]: Would have to be Item as ItemItem, else IM will try to get the entity "Item"

File: src/pocketmine/entity/Chicken.php [hunk 4]
```diff
@@ -24,44 +24,58 @@
...
-	}
+    public function initEntity(){
+        $this->setMaxHealth(4);
+        if(isset($this->namedtag->Health)){
+            $this->setHealth((int) $this->namedtag["Health"]);
+        }else{
+            $this->setHealth($this->getMaxHealth());
+        }
+        parent::initEntity();
+        $this->created = true;
```
Comment (thebigsmileXD) [inline src/pocketmine/entity/Chicken.php]: I don't get it. that like is so useless

File: src/pocketmine/entity/Chicken.php [hunk 5]
```diff
@@ -24,44 +24,58 @@
...
+    }
 
-	public function spawnTo(Player $player){
-		$pk = $this->addEntityDataPacket($player);
-		$pk->type = Chicken::NETWORK_ID;
+    public function targetOption(Creature $creature, $distance){
```
Comment (thebigsmileXD) [inline src/pocketmine/entity/Chicken.php]: I like that function. Its in bukkit also.

File: src/pocketmine/entity/Chicken.php [hunk 6]
```diff
@@ -24,44 +24,58 @@
...
+        if(isset($this->namedtag->Health)){
+            $this->setHealth((int) $this->namedtag["Health"]);
+        }else{
+            $this->setHealth($this->getMaxHealth());
+        }
+        parent::initEntity();
+        $this->created = true;
+    }
 
-	public function spawnTo(Player $player){
```
Comment (thebigsmileXD) [inline src/pocketmine/entity/Chicken.php]: no spawnTo, no chicken!

**Comment by thebigsmileXD** [comment]
Really? so many issues .-. 0% thought about that? @erkam2002

**Pull request closed without merging**